cli/src/commands/test.ts

```diff
@@ -41,8 +43,8 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
       process.cwd(),
       programAddress,
       `${programName}.so`,
-      `yarn ts-mocha -t 2000000 tests/${projectName}.ts --exit`
-    ]
+      `yarn ts-mocha -t 2000000 tests/${projectName}.ts --exit`,
+    ],
```

Comment:
are we sure this isn't a diverging configuration (will other folks' PRs revert formatting changes like this one)?

Reply:
the prettier formats it automatically according the latest formatting standard

Reply:
the test files were not part of the format command prior to this pr, thus it's possible that they were not formatted correctly before

Reply:
definitely.